spec/suites/layer/GeoJSONSpec.js

```diff
@@ -512,3 +512,364 @@ describe("L.LayerGroup#toGeoJSON", function () {
+					type: "invalid",
+					coordinates: [0, 0]
+				}
+			}).to.throwError("Invalid GeoJSON object.");
+		});
+
+		it("returns nothing if feature does not have a geometry property", function () {
+			var ret = L.GeoJSON.geometryToLayer({type: "Feature"});
+			expect(ret).not.to.be.ok();
+		});
+
+		it("creates a Layer using pointToLayer option (Point)", function () {
+			var layer = L.GeoJSON.geometryToLayer({
+				type: "Feature",
+				geometry: point,
+				properties: {radius: 100}
+			}, {
+				pointToLayer: customPointToLayer
+			});
+			expect(layer).to.be.a(L.Circle);
+			expect(layer.options.radius).to.be(100);
+		});
+
+		it("creates a Layer using pointToLayer option (MultiPoint)", function () {
+			var layer = L.GeoJSON.geometryToLayer({
+				type: "Feature",
+				geometry: multiPoint,
+				properties: {radius: 100}
+			}, {
+				pointToLayer: customPointToLayer
+			});
+			layer.eachLayer(function (lyr) {
+				expect(lyr).to.be.a(L.Circle);
+				expect(lyr.options.radius).to.be(100);
+			});
+		});
+
+		it("creates a Layer using coordsToLatLng option (Point)", function () {
+			var layer = L.GeoJSON.geometryToLayer({
+				type: "Feature",
+				geometry: {
+					type: "Point",
+					coordinates: [1, 2, 3]
+				}
+			}, {
+				coordsToLatLng: customCoordstoLatLng
+			});
+			expect(layer.getLatLng()).to.eql({lat: 3, lng: 2, alt: 4});
+		});
+
+		it("creates a Layer using coordsToLatLng option (MultiPoint)", function () {
+			var layer = L.GeoJSON.geometryToLayer({
+				type: "Feature",
+				geometry: {
+					type: "MultiPoint",
+					coordinates: [
+						[1, 2, 3], [4, 5, 6]
+					]
+				}
+			}, {
+				coordsToLatLng: customCoordstoLatLng
+			});
+			expect(layer.getLayers().map(function (lyr) {
+				return lyr.getLatLng();
+			})).to.eql([
+				{lat: 3, lng: 2, alt: 4},
+				{lat: 6, lng: 5, alt: 7}
+			]);
+		});
+	});
+
+	describe("#coordsToLatLng", function () {
+		it("creates a LatLng object", function () {
+			var latLng = L.GeoJSON.coordsToLatLng([1, 2]);
+			var latLngWithAlt = L.GeoJSON.coordsToLatLng([3, 4, 5]);
+			expect(latLng).to.be.a(L.LatLng);
+			expect(latLngWithAlt).to.be.a(L.LatLng);
+		});
+
+		it("creates a LatLng object with given coordinates", function () {
+			var latLng = L.GeoJSON.coordsToLatLng([1, 2]);
+			var latLngWithAlt = L.GeoJSON.coordsToLatLng([3, 4, 5]);
+			expect(latLng.lat).to.be(2);
+			expect(latLng.lng).to.be(1);
+			expect(latLng.alt).not.to.be.ok();
+			expect(latLngWithAlt.lat).to.be(4);
+			expect(latLngWithAlt.lng).to.be(3);
+			expect(latLngWithAlt.alt).to.be(5);
+		});
+	});
+
+	describe("#coordsToLatLngs", function () {
+
+		function customCoordsToLatLng(coords) {
+			return L.latLng(coords[1] + 1, coords[0] + 1, coords[2] + 1);
+		}
+
+		it("creates a multidimensional array of LatLngs", function () {
+			var latLngs = L.GeoJSON.coordsToLatLngs([[1, 2], [3, 4], [5, 6]]);
+			expect(latLngs.length).to.be(3);
+			latLngs.forEach(function (latLng) {
+				expect(latLng).to.be.a(L.LatLng);
```

Comment:
Better compare exact result with `eql`.

Reply:
The [last review](https://github.com/Leaflet/Leaflet/pull/7147#discussion_r429552996) you told me, i should use `to.be.a` in favor of `expect(latLng instanceof L.LatLng).to.be(true)` which i find reasonable, but i don't get why i should use `eql` because i am asserting against a type and find the test with `.to.be.a` more readable.

Reply:
> `.to.be.a` more readable.

It is more readable when you want to test a type of value only.
But here I request you not just test against a type, but also check exact values.
Well, you could do it in separate test, I suppose that it would be overkill.

Reply:
So i should not assert against the type and instead against the single values (coordinates)?

Reply:
Done with cc74c2c